test/exe/require_glibc.go

```diff
@@ -0,0 +1,39 @@
+package main
```

Comment:
Just one comment above this to explain the purpose and invocation format? "This program checks the glibc version is higher than 2.27. Example `./require_glibc <binary>`."